packages/tracing/src/integrations/postgres.ts

```diff
@@ -0,0 +1,81 @@
+import { Hub } from '@sentry/hub';
+import { EventProcessor, Integration } from '@sentry/types';
+import { fill, logger } from '@sentry/utils';
+
+interface PgClient {
+  prototype: {
+    query: () => void | Promise<unknown>;
+  };
+}
+
+interface PostgresOptions {
+  client?: PgClient;
+}
+
+/** Tracing integration for node-postgres package */
+export class Postgres implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'Postgres';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = Postgres.id;
+
+  private _client?: PgClient;
+
+  /**
+   * @inheritDoc
+   */
+  public constructor(options: PostgresOptions = {}) {
+    this._client = options.client;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setupOnce(_: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
+    if (!this._client) {
+      logger.error('PostgresIntegration is missing a Postgres.Client constructor');
```

Comment:
I'm a little confused here. Is `this._client` a class instance or a class itself? `private _client?: PgClient;` implies the former while `logger.error('PostgresIntegration is missing a Postgres.Client constructor');` implies the latter.

EDIT: Oh, I get it - the interface above has `prototype`, so it's meant to be class. Could we call it `PgClientClass` to make that more obvious?